recipe/meta.yaml

```diff
@@ -11,31 +11,33 @@ source:
 
 build:
   noarch: python
-  number: 4
+  number: 5
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ stdlib('c') }}
-    - python >=3.8
-    - numpy
     - cmake
-    - make  # [not win]
+    - ninja
   host:
-    - python >=3.8
+    - python 3.10
     - eigenpy
-    - pinocchio >=2,<4
+    - pinocchio
     - numpy
   run:
-    - python >=3.8
+    - python >={{ python_min }}
     - numpy
-    - pinocchio >=2,<4
+    - pinocchio
+    - __unix  # [unix]
+    - __win   # [win]
 
 test:
   requires:
     - pkg-config
   commands:
     - pkg-config --modversion {{ name }}
+    - if not exist %LIBRARY_PREFIX%\\lib\\cmake\\example-robot-data\\example-robot-dataConfig.cmake exit 1  # [win]
```

Comment:
Don't think the double antislash is necessary here, but it should still work (see https://github.com/conda-forge/zlib-feedstock/blob/main/recipe/meta.yaml for example)